edxval/utils.py

```diff
@@ -185,3 +197,24 @@ def create_file_in_fs(file_data, file_name, file_system, static_dir):
     """
     with file_system.open(combine(static_dir, file_name), 'wb') as f:
         f.write(file_data)
+
+
+def get_transcript_format(transcript_content):
+    """
+    Returns transcript format.
+
+    Arguments:
+        transcript_content (str): Transcript file content.
+    """
+    try:
+        sjson_obj = json.loads(transcript_content)
+    except ValueError:
+        # With error handling (set to 'ERROR_RAISE'), we will be getting
+        # the exception if something went wrong in parsing the transcript.
+        srt_subs = SubRipFile.from_string(
+            transcript_content.decode('utf-8-sig'),
```

Comment:
What if we have `UnicodeDecodeError`? Also this should be done out side of this method -- one level above!